packages/PyTrilinos/src/Domi.i

```diff
@@ -531,10 +548,10 @@ MDMap = MDMap_default
 %ignore Domi::MDVector::getDataNonConst(bool includePadding = true);
 %ignore Domi::MDVector::getData(bool includePadding = true) const;
 %include "Domi_MDVector.hpp"
-%teuchos_rcp(Domi::MDVector< int      , Domi::DefaultNode::DefaultNodeType >)
-%teuchos_rcp(Domi::MDVector< long long, Domi::DefaultNode::DefaultNodeType >)
-%teuchos_rcp(Domi::MDVector< double   , Domi::DefaultNode::DefaultNodeType >)
-//%teuchos_rcp(Domi::MDVector< float    , Domi::DefaultNode::DefaultNodeType >)
+%teuchos_rcp(Domi::MDVector< int       >)
+%teuchos_rcp(Domi::MDVector< long long >)
```

Comment:
Does Domi require `Tpetra_INST_INT_LONG_LONG=ON`?  If not, then you can't use `Scalar=long long` with Tpetra.

Reply:
Theoretically it does, but it may not be implemented that way specifically.